frontend/src/components/Landing/PsychologistFinder.jsx

```diff
@@ -11,7 +11,7 @@ const PsychologistFinder = () => {
       <img
         className={styles.image}
         src={`/images/illustration_sante_psy${theme === 'dark' ? '_dm' : ''}.png`}
-        alt="Trouver un psychologue"
+        alt="" // decorative image
```

Comment:
why don't you put the decorative image inside ?

Reply:
We shouldn't use `alt` for decorative image. 
Alt value is something we can read if we can't see the page. There is no point reading a decorative image for someone who can't see the page. That's why I removed it and added a comment to avoid adding it in the future.